public/python.py

```diff
@@ -0,0 +1,10 @@
+import sys
+import json
+
+# expects sys.argv[1] to be path for json file, formatted as per graasp research
+with open(sys.argv[1]) as json_file:
+    allData = json.load(json_file)
+    with open("anonymized_actions.json", "w+") as anonymizedFile:
```

Comment:
Where does this file comes from? 
I think to be less dependent on scripts, assure saving the files each time, and be able to list the resulting file in the database later, it will be necessary to save the file in our back-end (instead of in the script). If possible, can you return/print the result content in the python script, and get it back in the js backend to save it locally in `VAR_FOLDER`?

Reply:
@pyphilia I will create a separate issue for this - would be great if you can show how it's done.